<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="statusBar_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>811</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter un recycleur</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>671</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string>AJOUT</string>
      </property>
      <widget class="QLabel" name="label_id">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>identifiant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_np">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom et prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ad">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_numtel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>telephone</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5typem">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>type de materiaux</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>150</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adresse">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_tele">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>270</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_type">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>340</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>210</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>AJOUTER</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>MODIFIER</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>afficher les recyleurs </string>
     </attribute>
     <widget class="QTableView" name="tab_recycleur">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>621</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>supprimer un recycleur</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>tapez le nom du recycleur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supp_nom">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>chercher</string>
     </attribute>
     <widget class="QLineEdit" name="le_id_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_chercher">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/image/image/valider.png</normaloff>:/image/image/valider.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="id_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CHERCHER SUIVANT ID</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_trie">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>nom_prenom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type_materiax</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_trierNom">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>100</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/image/image/trie.png</normaloff>:/image/image/trie.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TRI SUIVANT LE NOM </string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_rec">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>331</width>
        <height>211</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="PDF_pb">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>410</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 10px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images_espace/Images_espace/pdf.png</normaloff>:/images_espace/Images_espace/pdf.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter en PDF</string>
      </property>
     </widget>
     <widget class="QTableView" name="tabrech">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>351</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>login</string>
     </attribute>
     <widget class="QFrame" name="frame_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit_nom_2">
       <property name="geometry">
        <rect>
         <x>275</x>
         <y>279</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Nom...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenom_2">
       <property name="geometry">
        <rect>
         <x>274</x>
         <y>204</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Prenom...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_email_2">
       <property name="geometry">
        <rect>
         <x>273</x>
         <y>130</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Username...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_motdepasse">
       <property name="geometry">
        <rect>
         <x>273</x>
         <y>349</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Mote de Passe...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_log_in">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>608</x>
         <y>194</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>log in</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>607</x>
         <y>264</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>119</x>
         <y>206</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>280</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>87</x>
         <y>350</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Mot de Passe</string>
       </property>
      </widget>
      <widget class="QComboBox" name="quest">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>410</y>
         <width>461</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Choisir une question</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quel est le nom de votre premier chien?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quel est votre surnom?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>quel est le nom de votre meilleur ami?</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="reponse">
       <property name="geometry">
        <rect>
         <x>493</x>
         <y>414</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Reponse...</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>staistique</string>
     </attribute>
     <widget class="QCustomPlot" name="gplot" native="true">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>601</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>sati</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>arduino</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
